daphne_worker/src/durable/reports_processed.rs

```diff
@@ -86,4 +99,9 @@ impl DurableObject for ReportsProcessed {
             ))),
         }
     }
+
+    async fn alarm(&mut self) -> Result<Response> {
+        self.state.storage().delete_all().await?;
```

Comment:
After the alarm fires, the DO may still be in memory. Thus we need to clear its volatile state as well.
```suggestion
        self.state.storage().delete_all().await?;
        self.touched = false;
        self.alarmed = false;
```